src/main/java/com/example/mate/controller/BookController.java

```diff
@@ -26,6 +27,7 @@ public List<BookDto> getAll() {
     }
 
     @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Why NotFound? you should return a response with OK

Reply:
In this place, I made a mistake because I removed this code from 
the EntityNotFoundException class and moved it to the getBookById method,
but I forgot to change the status to @ResponseStatus(HttpStatus.OK).